network/manager.go

```diff
@@ -698,3 +687,54 @@ func (nm *networkManager) GetEndpointID(containerID, ifName string) string {
 	}
 	return containerID + "-" + ifName
 }
+
+func cnsEndpointInfotoCNIEpInfo(endpointInfo restserver.EndpointInfo, endpointID string) *EndpointInfo {
+	epInfo := &EndpointInfo{
+		Id:                  endpointID,
+		IfIndex:             EndpointIfIndex, // Azure CNI supports only one interface
+		ContainerID:         endpointID,
+		PODName:             endpointInfo.PodName,
+		PODNameSpace:        endpointInfo.PodNamespace,
+		NetworkContainerID:  endpointID,
+		SecondaryInterfaces: make(map[string]*InterfaceInfo),
+	}
+	// filling out the InfraNIC from the state
```

Comment:
could we move this comment to right above the if ifName == InfraInterfaceName?

Reply:
done